### DISCUSSION TITLE
Recommendations for configuring things like wifi, bluetooth, sound, monitors.

### DISCUSSION THREAD

**Discussion by ferdinandyb** [i3/i3#6341] [Q&A]
I've recently started migrating from [Regolith](https://regolith-desktop.com/) to [CachyOS](https://cachyos.org/), and while Regolith was built on top of Gnome and had the gnome settings app for things like picking wifi, bluetooth, monitor alignment, sound etc, CachyOS's i3 version is way more streamlined. I guess this means I sort of have to choose something for each of these tasks, but for example I'm not too keen on using xrandr on the CLI directly to set up my monitors. Ideally I'd want to have a keybinding launching a configurator for the most common things and I'm looking for recommendations for this.

**Comment by okraits** [comment]
* network settings: Network-Manager with nm-applet
* sound: pavucontrol
* monitor configuration: arandr

For bluetooth I can't give a recommendation because I don't use it.

**Reply by parz3val** [reply]
I use blueman for bluetooth. It works most of the time.

**Reply by ferdinandyb** [reply]
Thanks, those were all very good suggestions thanks! For blueman the "most of the time" is correct :D For pairing a keyboard I needed to manually connect in `bluetoothctl` (it complained that the keyboard does not have sound controls ...) which is not the world's best experience, but also fortunately not an every day issue.

**Reply by orestisfl** [reply]
I have a small script for connecting to my headphones on demand instead of fighting with blueman.

```bash
#!/bin/bash
systemctl is-active bluetooth || systemctl start bluetooth
bluetoothctl power on || { sleep 1 && bluetoothctl power on; }
bluetoothctl devices Trusted | grep "$DEVICE_NAME" | cut -d ' ' -f 2  | xargs bluetoothctl connect || blueman-manager&
```
